src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -192,10 +194,17 @@ export default class ReviewDataContent extends React.Component {
                                         Monthly submissions cannot be certified
                                     </div>
         }
-        if (this.checkAffiliations() || this.props.session.admin) {
+
+        if ((this.checkAffiliations() || this.props.session.admin ) && buttonClass != " btn-disabled") {
             certifyButtonText = "Certify & Publish the Submission to USAspending.gov";
             buttonClass = "";
-            buttonAction = this.openModal.bind(this, modalToOpen);;
+            buttonAction = this.openModal.bind(this, modalToOpen);
+        }
+
+        if(buttonClass != " btn-disabled" && this.props.data.gtas && this.props.data.gtas.data) {
```

Comment:
If you're already setting gtas to `res.data` you don't have to check for `gtas.data` (it'll always be empty actually because it doesn't exist).